src/commands/syncModel/run/README.md

```diff
@@ -1,101 +1,249 @@
+- **Deployment Automation**: Automate content model updates as part of your CI/CD pipeline.
+- **Team Collaboration**: Allow multiple team members to work on content models in different environments and synchronize changes efficiently.
+
+## Source Content Model Options
+
+You can obtain the source content model in two ways:
+
+1. **From an Existing Kontent.ai Environment**: Provide the source environment ID and Management API key to fetch the content model directly.
+2. **From a Local Folder**: Use a local folder containing the content model exported using the [`sync-model export`](../export/README.md) command.
+
+**Using a local snapshot of the content model is the recommended approach**, as it creates a stable snapshot that won't change during the synchronization process. This prevents discrepancies that can occur if changes are made in the source environment between diffing and syncing.
+
+## Key Features
+
+- **Selective Synchronization**: Specify which entities to synchronize using the mandatory `--entities` parameter.
+- **Pre-Sync Validation**: Performs validation checks to prevent operations that could cause errors or data loss.
+- **Change Preview (Diff)**: Generates and displays a diff of the changes before applying them, allowing you to review and confirm.
+
+## Supported Entities
+
+The following entities can be synchronized:
+
+- **Content Types**
+- **Content Type Snippets**
+- **Taxonomies**
+- **Web Spotlight**
+- **Asset Folders**
+- **Collections**
+- **Spaces**
+- **Languages**
+- **Workflows**
+
+## Important Considerations
+
+- **Entity Matching**: Entities are matched by their `codename`.
+- **Partial Synchronization**: Ensure all dependent entities are included or already exist in the target environment.
+- **No External ID Synchronization**: `external_id` properties are not synchronized to avoid conflicts.
+- **Guidelines References**: References to items or assets not present in the target environment will use `external_id` after synchronization.
+
+## Conditions for Successful Synchronization
+
+Before running the synchronization, ensure:
+
+- **No Element Type Changes**: Changing the type of an existing element is not allowed.
+- **No Deletion of Used Content Types**: Cannot delete content types with existing content items.
+- **No Deletion of Used Collections**: Cannot delete collections containing content items.
+- **No References to Deleted Taxonomies**: The source model should not reference deleted taxonomies.
+- **Valid Content Model**: If using a folder, ensure the content model files are valid and correctly structured.
+
+## Known Limitations
+
+- **Snippet Elements**: Cannot reference snippet elements in the same request they're created.
+- **Asset Folders**:
+  - Cannot be moved; they are deleted and recreated if structure differs.
+  - Cannot be deleted if containing assets; ensure folders to be deleted are empty.
```

Comment:
original was having the crucial information

Reply:
? what is missing that is in the original ? 
This is just a long version of basically the same info: Asset folders cannot be deleted (or moved, see the previous point) if they contain assets. The command is not able to check this without loading all the assets in the project so it doesn't check it. Please, make sure that all folders that will be deleted does not contain any assets. You can see what folders will be deleted in the generated diff sync-model diff ....